bgpd/bgpd.c

```diff
@@ -1731,9 +1731,8 @@ struct peer *peer_create(union sockunion *su, const char *conf_if,
 	peer->local_id = bgp->router_id;
 	peer->v_holdtime = bgp->default_holdtime;
 	peer->v_keepalive = bgp->default_keepalive;
-	peer->v_routeadv = (peer_sort(peer) == BGP_PEER_IBGP)
-				   ? BGP_DEFAULT_IBGP_ROUTEADV
-				   : BGP_DEFAULT_EBGP_ROUTEADV;
+	peer_sort(peer);
```

Comment:
Can you cast this to `(void)peer_sort(peer)` just to avoid another one coverity fix?

Reply:
Done.